code/utils/xcmp/xcm-examples.md

```diff
@@ -0,0 +1,27 @@
+# Overview
+
+List of useful examples of XCM messages for testing and playbooks.
```

Comment:
nit: It is up to you, but I generally prefer adding a link to the term so that newcomers who try to skim through our docs can easily check out them. I mean maybe we can add a link to `XCM`.